src/TodoList.js

```diff
@@ -1,30 +1,11 @@
 import React from 'react';
 import TodoListItem from './TodoListItem';
 
-function TodoList() {
-  const todoList = [
-    {
-      id: 1234,
-      title: 'Complete CTD assignment',
-    },
-    {
-      id: 4321,
-      title: 'Go grocery shopping',
-    },
-    {
-      id: 9876,
-      title: 'Go the gym',
-    },
-    {
-      id: 4987,
-      title: 'Do laundry',
-    },
-  ];
-
+function TodoList({ todoList }) {
   return (
     <ul>
-      {todoList.map((item) => (
-        <TodoListItem key={item.id} todo={item.title} />
+      {todoList.map(({ id, title }) => (
```

Comment:
Nice to see you adding in more destructuring here that I don't believe I saw in the instructions. Good work!